manual_flagging/flagging.py

```diff
@@ -57,6 +57,15 @@
         BETWEEN DATE '{date_floor}'
         AND DATE '{inputs['time_frame']['end']}')"""
 
+# Parse run_date and make sure its format is correct
+date_fmt = "%Y-%m-%d"
+run_date = (
```

Comment:
I added a note on what's going on with `run_date` to its definition in the `inputs.yml` file below.